CONTRIBUTING.md

```diff
@@ -262,6 +262,125 @@ or
 
 ``./test_infra/scripts/delete-stack.sh databases``
 
+## Ray Load Tests Environment 
+**DISCLAIMER**: Make sure you know what you are doing. These steps will charge some services on your AWS account and require a minimum security skill to keep your environment safe.
+
+* Pick up a Linux or MacOS.
+* Install Python 3.7, 3.8 or 3.9 with [poetry](https://github.com/python-poetry/poetry) for package management
+* Fork the AWS SDK for pandas repository and clone that into your development environment
+* Go to the project's directory create a Python's virtual environment for the project
+
+`python -m venv .venv && source .venv/bin/activate`
+
+* Then run the command bellow to install all dependencies:
+
+``poetry install``
+
+* Go to the ``test_infra`` directory
+
+``cd test_infra``
+
+* Install CDK dependencies:
+
+``poetry install``
+
+* [OPTIONAL] Set AWS_DEFAULT_REGION to define the region the Ray Test environment will deploy into. You may want to choose a region which you don't currently use:
+
+``export AWS_DEFAULT_REGION=ap-northeast-1``
+
+* Go to the ``scripts`` directory
+
+``cd scripts``
+
+* Deploy the `ray` CDK stack.
+
+``./deploy-stack.sh ray``
+
+* Configure Ray Cluster 
+
+``vi ray-cluster-config.yaml`` 
+
+```
+# Update the following file to match your enviroment
+# The following is an example
+cluster_name: ray-cluster
+
+initial_workers: 2
+min_workers: 2
+max_workers: 2
+
+provider:
+    type: aws
+    region: us-east-1 # change region as required
+    availability_zone: us-east-1a,us-east-1b,us-east-1c # change azs as required
+    security_group:
+        GroupName: ray_client_security_group
+    cache_stopped_nodes: False
+
+available_node_types:
+  ray.head.default:
+    node_config:
+      InstanceType: r5n.2xlarge # change instance type as required
+      IamInstanceProfile:
+        Arn: arn:aws:iam::{UPDATE YOUR ACCOUNT ID HERE}:instance-profile/ray-cluster-instance-profile
+      ImageId: ami-0ea510fcb67686b48 # latest ray images -> https://github.com/amzn/amazon-ray#amazon-ray-images 
+      NetworkInterfaces:
+        - AssociatePublicIpAddress: True
+          SubnetId: {replace with subnet within above AZs}
+          Groups: [{ID of group `ray_client_security_group` created by the step above}]
+          DeviceIndex: 0
+
+  ray.worker.default:
+      min_workers: 2
+      max_workers: 2
+      node_config:
+        InstanceType: r5n.2xlarge
+        IamInstanceProfile:
+          Arn: arn:aws:iam::{UPDATE YOUR ACCOUNT ID HERE}:instance-profile/ray-cluster-instance-profile
+        ImageId: ami-0ea510fcb67686b48 # latest ray images -> https://github.com/amzn/amazon-ray#amazon-ray-images 
+        NetworkInterfaces:
+          - AssociatePublicIpAddress: True
+            SubnetId: {replace with subnet within above AZs}
+            Groups: [{ID of group `ray_client_security_group` created by the step above}]
+            DeviceIndex: 0
+
+setup_commands:
+- pip install awswrangler==3.0.0a2
```

Comment:
I would just put `pip install 'awswrangler==3.0.0a2[distributed]'` which would install ray and modin too. We can then remove the subsequent two pip install commands for ray and modin. I am just worried we will forget about these lines and the versions will eventually be obsolete given that they are hardcoded